### PR TITLE
community minutes: 2023-04-06

### PR DESCRIPTION
**Pull request by tibeer**
Signed-off-by: Tim Beermann <beermann@osism.tech>

### CODE REVIEW

File: community-call/20230406.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+## Meta
+
+Minutes: @tibeer
+
+## Introduction of new Members
+
+- Sven from OSISM
+    - @artificial-intelligence
+- Thomas Güttler from SySelf
+    -
```
Comment (maxwolfs) [inline community-call/20230406.md:12]: remove empty line

**Review by maxwolfs — CHANGES_REQUESTED** [review]

File: community-call/20230406.md (hunk 1 above)
Comment (tibeer) [reply community-call/20230406.md]: Done.

**Review by maxwolfs — APPROVED** [review]
Thanks!

**Review by itrich — APPROVED** [review]
Thank you!

**Pull request merged**